### ISSUE TITLE
[1.17.1] server commands not working

### ISSUE THREAD

**Issue by Oni-Feralis** [lorddusk/HQM#596]
I'm currently attempting to get a questline set up for a friends fabric server, but the hqm commands will not run.
The book is spawnable/craftable, but has no quests, and I cannot use the in game editor because the command to begin/end editing is not running. However the command to give the op book to a player works, and the hearts work to regain lives.

Players are even getting the message that HQM is enabled, and they have 3 lives left, but I have /kill'd myself 4 times to make sure; the book contradicts this saying i have died, but have unlimited lives.

**Comment by kirderf1** [comment]
To better understand what the issue is:
- In what sense does the commands not work? What feedback are you getting?
- Do you have syncronization turned on in the HQM config on the server?

**Comment by Oni-Feralis** [comment]
When I said the commands are not working, they are not sending a response, like the /hqm version does not send a text response, and there is nothing on the console. and /hqm edit does not enable edit mode.

Server_sync is not turned on, and we're turning it on now to check if that was the issue
edit: turning on server_sync did not change anything

**Comment by kirderf1** [comment]
Unusual to not get any response. Are other parts of the command system working? (Such as autocomplete, getting an error from nonexistent commands, and running vanilla commands)
Also, do you think you could check your server logs (and I guess it can't hurt with logs from a player who joins the server) for any potential exceptions/stacktraces/error messages. If that fails, I think the next best is to have the mod updated with logging messages in the proper places to identify in which step that the server data sync fails

**Comment by Oni-Feralis** [comment]
so, apparently on serverside, the messages will not appear if an operator run the commands. I checked the console, and they are logging correctly(I didnt have access to console logs before). now that I have console commands, I ran "/hqm edit" and got the response, "Editing mode does not function  properly on servers, and has been disabled" on the console, but the operators did not get such a response.
